cloud/aws/modules/setup/secrets.tf

```diff
@@ -8,9 +8,8 @@ resource "aws_kms_key" "civiform_kms_key" {
 
 # Create a random generated password to use for postgres_password.
 resource "random_password" "postgres_username" {
-  length           = 16
-  special          = true
-  override_special = "_%@"
```

Comment:
Is this intentional?

Reply:
yes. we don't want special characters in username.